common/web/gesture-recognizer/src/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file should also be in component root, not `src/` folder. Next to its `package.json` file :smile: